Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" },
+        { "name": "EventState", "type": "ObjGravityCollisionSpawner::EventState" }
+      ]
+    },
+    "ObjGravityMachineSpawner": {
+      "fields": [
+        { "name": "RoundRadius", "type": "float32" },
+        { "name": "RoundSpeed", "type": "float32" },
+        { "name": "ShieldSpeed", "type": "float32" }
+      ]
+    },
+    "ObjHatchSpawner": {
+      "fields": [
+        { "name": "Type", "type": "ObjHatchSpawner::Type" },
+        { "name": "speed", "type": "float32" }
+      ]
+    },
+    "ObjLightTimeFloorSpawner": {
+      "fields": [
+        { "name": "OnFloorTime", "type": "float32" },
+        { "name": "ForetasteTime", "type": "float32" },
+        { "name": "AriseTime", "type": "float32" },
+        { "name": "Phase", "type": "float32" },
+        { "name": "FloorSize", "type": "ObjLightTimeFloorSpawner::FloorSize" },
+        { "name": "ActionType", "type": "ObjLightTimeFloorSpawner::ActionType" },
+        { "name": "EventType", "type": "ObjLightTimeFloorSpawner::EventType" }
+      ]
+    },
+    "ObjLongTankSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjMeteorSpawner": {
+      "fields": [
+        { "name": "SizeType", "type": "ObjMeteorSpawner::SizeType" }
+      ]
+    },
+    "ObjPotSpawner": {
+      "fields": [
+        { "name": "UpSpeed", "type": "float32" },
+        { "name": "SpeedMin", "type": "float32" },
+        { "name": "SpeedMax", "type": "float32" }
+      ]
+    },
+    "ObjPushBlockSpawner": {
+      "fields": [
+        { "name": "AppearTime", "type": "float32" },
+        { "name": "DisappearTime", "type": "float32" },
+        { "name": "AppearStopTime", "type": "float32" },
+        { "name": "DisappearStopTime", "type": "float32" },
+        { "name": "Phase", "type": "float32" },
+        { "name": "WaitTime", "type": "float32" },
+        { "name": "AppearDistance", "type": "float32" },
+        { "name": "BlockType", "type": "ObjPushBlockSpawner::BlockType" },
+        { "name": "IsEventStart", "type": "bool" },
+        { "name": "IsPushDead", "type": "bool" }
+      ]
+    },
+    "ObjRollerCoasterSpawner": {
+      "fields": [
+        { "name": "PathID", "type": "uint32" },
+        { "name": "StartPosition", "type": "float32" },
+        { "name": "MaxSpped", "type": "float32" },
+        { "name": "FristSpeed", "type": "float32" },
+        { "name": "Accel", "type": "float32" },
+        { "name": "VehicleNum", "type": "uint32" },
+        { "name": "Type", "type": "ObjRollerCoasterSpawner::Type" },
+        { "name": "PrepareSpeed", "type": "float32" }
+      ]
+    },
+    "ObjRollerCoasterCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" }
+      ]
+    },
```

Comment:
Unused object, appears in `stg901` of the Test Levels left over in Sonic Colors: Ultimate. Parameter names based on in-game functionality and it being a collision object.